### PR TITLE
Update disaster-recovery-azure-applications.md

### PR DESCRIPTION
**Pull request by EdPrice-MSFT**
Making edits from RoAnn and adding a few content pieces from David Read. Changes are only made to two sections: "Multiple datacenter regions" (minor edit and added the last sentence with the link) and "Data strategies for disaster recovery" (minor edits, added sub-headers to the "Backup and restore" section, and added the "Other data platforms" sub-section paragraph).

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resiliency/disaster-recovery-azure-applications.md](https://github.com/mspnp/architecture-center/blob/EdPrice-MSFT-patch-1/docs/resiliency/disaster-recovery-azure-applications.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resiliency/disaster-recovery-azure-applications?branch=pr-en-us-110) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C11%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705112047036075-110%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resiliency/disaster-recovery-azure-applications.md [hunk 1]
```diff
@@ -28,7 +29,7 @@ When you're configuring Traffic Manager, you provide a new Traffic Manager DNS p
 
 Within Traffic Manager, you specify the priority of the deployments that users will be routed to when failure occurs. Traffic Manager monitors the endpoints of the deployments and notes when the primary deployment fails. At failure, Traffic Manager analyzes the prioritized list of deployments and routes users to the next one on the list.
 
-Although Traffic Manager decides where to go in a failover, you can decide whether your failover domain is dormant or active while you're not in failover mode. That functionality has nothing to do with Azure Traffic Manager. Traffic Manager detects a failure in the primary site and rolls over to the failover site. Traffic Manager rolls over regardless of whether that site is currently serving users or not.
+dAlthough Traffic Manager decides where to go in a failover, you can decide whether your failover domain is dormant or active while you're not in failover mode. That functionality has nothing to do with Azure Traffic Manager. Traffic Manager detects a failure in the primary site and rolls over to the failover site. Traffic Manager rolls over regardless of whether that site is currently serving users or not.
```
Comment (bennage) [inline docs/resiliency/disaster-recovery-azure-applications.md]: There are random `d` characters in several places.

**Comment by bennage** [comment]
There are lots of odd formatting issues in the PR. The new content seems to be short and self-contained. @alexbuckgit is in the process of reviewing and consolidating the resiliency content. This is something he should probably handle.
@MikeWasson, are you tracking this?

**Comment by MikeWasson** [comment]
@bennage  Alex is on it

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resiliency/disaster-recovery-azure-applications.md](https://github.com/mspnp/architecture-center/blob/EdPrice-MSFT-patch-1/docs/resiliency/disaster-recovery-azure-applications.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resiliency/disaster-recovery-azure-applications?branch=pr-en-us-110) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C12%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705121949131377-110%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resiliency/disaster-recovery-azure-applications.md](https://github.com/mspnp/architecture-center/blob/EdPrice-MSFT-patch-1/docs/resiliency/disaster-recovery-azure-applications.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resiliency/disaster-recovery-azure-applications?branch=pr-en-us-110) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C12%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705121952216292-110%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resiliency/disaster-recovery-azure-applications.md](https://github.com/mspnp/architecture-center/blob/EdPrice-MSFT-patch-1/docs/resiliency/disaster-recovery-azure-applications.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resiliency/disaster-recovery-azure-applications?branch=pr-en-us-110) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C12%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705122107502776-110%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**